Doc/library/functools.rst

```diff
@@ -535,6 +535,8 @@ The :mod:`functools` module defines the following functions:
    .. versionchanged:: 3.7
       The :func:`register` attribute now supports using type annotations.
 
+   .. versionchanged:: 3.11
+      Implementation functions are now registered using :func:`register_variant`.
```

Comment:
This feels a little opaque to me (same for the notice on line 592). Maybe we should wait until we've taught `inspect.signature()` to somehow use `get_variants` to give better results for multi-dispatch functions before we add these `.. versionchanged` notices? I feel like the benefit to the end user will be easier to explain at that point.

Reply:
I made it reference `get_variants` now, which is a bit more immediately useful to users.